### ISSUE TITLE
Change the name of the generated trino tables

### ISSUE THREAD

**Issue by AndreasTA-AW** [metriql/metriql#47]
Hi,

is it today possible to change how the name of the tables gets generated. We would probably prefer to leave out the model_<dbt-project> path of it.

**Comment by buremba** [comment]
In dbt, the models and sources are not unique so we had to have these prefixes to avoid name collisions. However; we support dbt references so you can access them as follows:

```
select count(*) from "ref('orders')" 
```

or:

```
select count(*) from "source('tpch', 'orders')" 
```

Which tool are you using? Maybe we should expose the sources as dbt references, not internal names?

**Comment by AndreasTA-AW** [comment]
That is true. I explored it a bit with DBeaver now and it doesn't feel super nice with the naming convention currently popping up :) For eg. we only expose a virtual mart to metriql, or for that case all the places actually consuming data, so sources are never exposed outside of DBT. It wouldn't be possible to just give a naming convention for sources so it turns out to be tpch_orders? I feel there must be some sweet solution :)

One more reason is if there is no "automation" ready for a BI Tool, the models(or whatever it's called in each BI Tool) will have really weird names for the end user, instead of actually just showing them what they are looking for.
